slices/stunnel4.yaml

```diff
@@ -0,0 +1,30 @@
+package: stunnel4
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libc6_config
+      - libssl3_libs
+      - openssl_config
+      - libgssapi-krb5-2_libs
+      - libtirpc3_libs
+      - libgpg-error0_libs
+      - libcom-err2_libs
+      - liblzma5_libs
+      - libcap2_libs
+      - libgpg-error0_libs
+      - libkeyutils1_libs
+      - liblz4-1_libs
+      - libsystemd0_libs
+      - libwrap0_libs
+      - libzstd1_libs
+      - libgcrypt20_libs
+      - libnsl2_libs
+      - libgcc-s1_libs
+    contents:
+      /usr/bin/stunnel3:
```

Comment:
This is something I believe we should be careful about. I'm not too familiar with the inner workings of stunnel, but do you actually need both binaries? I.e., if these are complementary binaries that can work independently of each other, then I'd recommend splitting them into dedicated slices, such that users are not forced to have both when they actually only need one.

We call this "slicing by functional unit". Basically, instead of slicing a package by the type of files it contains, we design its slices with basis on the functionality we're looking for. See an example here: https://github.com/canonical/chisel-releases/blob/ubuntu-22.04/slices/openjdk-8-jre-headless.yaml